pilot/pkg/bootstrap/options.go

```diff
@@ -123,7 +123,7 @@ type MCPOptions struct {
 	InitialConnWindowSize int
 }
 
-var PodNamespaceVar = env.RegisterStringVar("POD_NAMESPACE", "", "")
+var PodNamespaceVar = env.RegisterStringVar("POD_NAMESPACE", "istio-system", "")
```

Comment:
For consistency, how about setting accurate default values for all of them ? POD_NAME=istiod, etc.

And while doing this - also set the default value for "-n" - which is empty but should be istio-system.

(maybe all in a separate PR :-)

Reply:
Pod name has no default, it includes the full suffix. Ill do the namespace one in another PR

Reply:
the motivation for this part was the 50 line yaml demo I have for this, by the way